andela/wg-leo
# [Feature 158626060] Reset muscle cache on delete

**What does this PR do?**
Resets cache when deleting muscles

**Description of Task completed**
Initially, after deleting a muscle, it still appeared under the exercise description associated with deleted muscle. This was caused by a failure to reset muscle cache on delete.
The cache should be reset after a muscle has been deleted such that the deleted muscle in not visible from the exercises page.
Files changed include:

 `wger/exercises/signals.py`

**How should this be manually tested?**
-Log in to the application as an admin, visit the muscle menu under the exercise tab 
https://wgerl-staging.herokuapp.com/en/exercise/muscle/admin-overview/
-Click on the add muscle button and add a muscle name.
-Visit the add exercise tab and add a new exercise focusing on the new muscle added.
-Go back to the muscles menu and delete the recently added muscle and verify that the muscle no-longer included under the exercise menu.


**What are the relevant pivotal tracker stories?**
[#158526060](https://www.pivotaltracker.com/n/projects/2181164)
